### PR TITLE
Add lab permissions

### PR DESCRIPTION
**Pull request by dmcmanam**

### CODE REVIEW

File: src/test/java/org/mskcc/utils/ParserUtilTest.java [hunk 1]
```diff
@@ -1,93 +1,93 @@
...
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringRunner;
-
-import java.util.List;
-
-@SpringBootTest
-@RunWith(SpringRunner.class)
-public class ParserUtilTest {
```
Comment (DavidStreid) [inline src/test/java/org/mskcc/utils/ParserUtilTest.java:13]: I checked out and uncommented - these succeed. Can we add them back in?

**Review by DavidStreid — CHANGES_REQUESTED** [review]
Looks good! Can we add back the `ParserUtilTest.java` though?

And should we be running w/ these JVM options `-Xmx64m" "-Xms64m`? I saw that they were added by default, but I don't think they're currently being used.

File: src/test/java/org/mskcc/utils/ParserUtilTest.java (hunk 1 above)
Comment (dmcmanam) [reply src/test/java/org/mskcc/utils/ParserUtilTest.java]: I should not have committed the commented out class.  It is back.

File: gradlew [hunk 2]
```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```
[the remark this answers is not in the document]
Comment (dmcmanam) [reply gradlew:47]: I'm not sure why those changes are there, "gradle wrapper" generated them.

**Pull request merged**